app/src/main/res/values-de/strings.xml

```diff
@@ -75,31 +75,31 @@
     <string name="action_start">Start</string>
     <string name="action_stop">Stop</string>
     <string name="automatic_refresh">Täglich aktualisieren</string>
-    <string name="automatic_refresh_description">Tägliche Aktualisierungen finden statt, wenn das Gerät lädt, in Ruhe ist, und die Verbindung gebührenfrei ist.</string>
+    <string name="automatic_refresh_description">Tägliche Aktualisierungen, wenn das Gerät geladen wird, im Ruhezustand ist und die Verbindung gebührenfrei ist.</string>
     <string name="watchdog">Überwache Verbindung</string>
-    <string name="watchdog_description">Prüfe die Verbindung in ansteigenden Abständen und verbinde erneut, falls sie nicht mehr funktioniert.</string>
-    <string name="legend_host_intro">Einträge werden der Reihenfolge nach bearbeitet. Enthalten mehrere den gleichen Host, so wird der letzte Angewandt. Die Aktionen sind:</string>
+    <string name="watchdog_description">Prüfe die Verbindung in regelmäßigen Abständen und verbinde erneut, falls diese nicht mehr funktioniert.</string>
```

Comment:
Hat nichts mit akkurat zu tun eher was mit Verständnis, es gibt keine App die 'ansteigend' nutzt, in keiner mir bekannten Firewall oder DNS blocker app.